### PR TITLE
doc: cleaning up the language in Power optimization doc

### PR DESCRIPTION
**Pull request by peknis**
Making the tone more active and some other fixes.

Signed-off-by: Pekka Niskanen <pekka.niskanen@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/app_power_opt.rst [hunk 1]
```diff
@@ -127,104 +133,111 @@ To simulate the use case with Online Power Profiler, complete the following step
...
+#. Before tuning, calculate the sending frequency of the payload, with the targeted battery life of two years:
 
    .. code-block:: none
 
       139.9 mC / (57-4.6) µA = 44.5 minutes (4.6 µA is the measured PSM floor)
 
-   It is recommended to send a batch of six measurements for every 60 minutes to have some margin.
-   Hence the payload size can be changed to a value of 120 bytes in the :file:`prj.conf` to observe how it affects the charge in a single transmission.
+   Send a batch of six measurements for every 60 minutes to have some margin.
+   Hence, you can change the payload size to a value of 120 bytes in the :file:`prj.conf` to observe how it affects the charge in a single transmission.
```
Comment (umapraseeda) [inline doc/nrf/app_power_opt.rst:214]:
```suggestion
    It is recommended to send a batch of six measurements for every 60 minutes to have some margin.
   Hence, you can change the payload size to a value of 120 bytes in the :file:`prj.conf` to observe how it affects the charge in a single transmission.
```

File: doc/nrf/app_power_opt.rst [hunk 2]
```diff
@@ -127,104 +133,111 @@ To simulate the use case with Online Power Profiler, complete the following step
...
+   A. In the :guilabel:`PSM` tab on the left, set :guilabel:`Active time (sec)` to ``0`` (indicating that the device need not be active after sending the payload).
+   #. In the :guilabel:`Data transfer` tab on the left, select ``Upload`` from the drop-down menu for :guilabel:`Enable`. The :guilabel:`Data size` and :guilabel:`Data transfer interval` options are now visible. Set the options to the following values:
 
-     a. Set :guilabel:`Data size (bytes)` to ``20``
-     #. Set :guilabel:`Data transfer interval (sec)` to ``600`` (10 minutes)
+     a. Set :guilabel:`Data size (bytes)` to ``20``.
+     #. Set :guilabel:`Data transfer interval (sec)` to ``600`` (10 minutes).
 
-Below is the result of the power profile simulation using Online Power Profiler:
+	The result of the power profile simulation using Online Power Profiler:
```
Comment (umapraseeda) [inline doc/nrf/app_power_opt.rst:142]:
```suggestion
The following image shows the result of the power profile simulation using Online Power Profiler:
```

File: doc/nrf/app_power_opt.rst [hunk 3]
```diff
@@ -127,104 +133,111 @@ To simulate the use case with Online Power Profiler, complete the following step
...
       :alt: Power Profiler Kit II sample output for 40-byte payload
 
-   Observe that the whole sequence of data transmission spans 12.32 seconds and the process consumes a total battery charge of 139.9 mC.
+   The whole sequence of data transmission spans now 12.32 seconds and the process consumes a total battery charge of 139.9 mC.
    According to the Online Power Profiler estimation, the time duration available for the data transmission sequence is 8.34 seconds with a battery charge of 53.94 mC.
    The longer data transmission duration in the real-time measurement using Power Profiler Kit II might be due to the RRC inactivity timer network parameter.
-   Hence it is recommended to tune the parameters to match the network environment.
-#. Before performing the tuning, you must calculate the frequency with which the payload can be sent, with the targeted battery life of two years.
+   Hence, tune the parameters to match the network environment.
+#. Before tuning, calculate the sending frequency of the payload, with the targeted battery life of two years:
```
Comment (umapraseeda) [inline doc/nrf/app_power_opt.rst:207]:
```suggestion
#. Before tuning, calculate the payload transmission frequency, with the targeted battery life of two years:
```

**Review by umapraseeda — CHANGES_REQUESTED** [review]

File: doc/nrf/app_power_opt.rst [hunk 4]
```diff
@@ -8,42 +8,46 @@ Power optimization
...
 
 Nordic System-on-Chip (SoC) designs are aimed at ease of use and ultra-low power consumption.
-The eco system consisting of developer tools like |NCS|, the development kits, the `Online Power Profiler (OPP)`_ and the `Power Profiler Kit II (PPK2)`_ provides a unified solution to the developers to evaluate, develop and characterize ultra-low power designs with ease.
-See :ref:`getting_started` for more information on |NCS| and the development environment.
+The development ecosystem consists of developer tools like:
+
+* |NCS|
+* The development kits
+* The `Online Power Profiler (OPP)`_
+* The `Power Profiler Kit II (PPK2)`_
```
[the remark this answers is not in the document]
Comment (peknis) [reply doc/nrf/app_power_opt.rst]: Done

File: doc/nrf/app_power_opt.rst [hunk 5]
```diff
@@ -127,104 +133,111 @@ To simulate the use case with Online Power Profiler, complete the following step
...
+   A. In the :guilabel:`PSM` tab on the left, set :guilabel:`Active time (sec)` to ``0`` (indicating that the device need not be active after sending the payload).
+   #. In the :guilabel:`Data transfer` tab on the left, select ``Upload`` from the drop-down menu for :guilabel:`Enable`. The :guilabel:`Data size` and :guilabel:`Data transfer interval` options are now visible. Set the options to the following values:
 
-     a. Set :guilabel:`Data size (bytes)` to ``20``
-     #. Set :guilabel:`Data transfer interval (sec)` to ``600`` (10 minutes)
+     a. Set :guilabel:`Data size (bytes)` to ``20``.
+     #. Set :guilabel:`Data transfer interval (sec)` to ``600`` (10 minutes).
 
-Below is the result of the power profile simulation using Online Power Profiler:
+The result of the power profile simulation using Online Power Profiler:
```
[the remark this answers is not in the document]
Comment (peknis) [reply doc/nrf/app_power_opt.rst]: Done

File: doc/nrf/app_power_opt.rst (hunk 1 above)
Comment (peknis) [reply doc/nrf/app_power_opt.rst]: Done

File: doc/nrf/app_power_opt.rst (hunk 2 above)
Comment (peknis) [reply doc/nrf/app_power_opt.rst]: Done

File: doc/nrf/app_power_opt.rst (hunk 3 above)
Comment (peknis) [reply doc/nrf/app_power_opt.rst]: Done

**Review by umapraseeda — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Pull request merged**